monitors/SyntheticPegMonitor.js

```diff
@@ -0,0 +1,100 @@
+const { createFormatFunction, createEtherscanLinkMarkdown } = require("../common/FormattingUtils");
+
+class SyntheticPegMonitor {
+  /**
+   * @notice Constructs new Liquidator bot.
+   * @param {Object} logger Module used to send logs.
+   * @param {Object} uniswapPriceFeed Module used to query the current uniswap token price.
+   * @param {Object} cryptoWatchPriceFeed Module used to query the current crypto watch token price.
+   * @param {Object} [config] Contains fields with which constructor will attempt to override defaults.
+   */
+  constructor(logger, uniswapPriceFeed, cryptoWatchPriceFeed) {
```

Comment:
Missing the `config` param in the constructor

Reply:
shit. good catch. I added the parm in the tests but didn't pass it in. will fix.